packages/gatsby/package.json

```diff
@@ -47,23 +47,24 @@
     "copyfiles": "^2.1.1",
     "core-js": "^2.6.10",
     "cors": "^2.8.5",
-    "css-loader": "^1.0.0",
-    "debug": "^3.1.0",
-    "del": "^3.0.0",
-    "detect-port": "^1.2.1",
-    "devcert": "^1.0.0",
-    "dotenv": "^4.0.0",
-    "eslint": "^5.6.0",
-    "eslint-config-react-app": "^3.0.0",
-    "eslint-loader": "^2.1.0",
-    "eslint-plugin-flowtype": "^2.46.1",
-    "eslint-plugin-graphql": "^3.0.3",
-    "eslint-plugin-import": "^2.9.0",
-    "eslint-plugin-jsx-a11y": "^6.0.3",
-    "eslint-plugin-react": "^7.8.2",
-    "event-source-polyfill": "^1.0.5",
-    "express": "^4.16.3",
-    "express-graphql": "^0.7.1",
+    "css-loader": "^1.0.1",
+    "debug": "^3.2.6",
+    "del": "^5.1.0",
+    "detect-port": "^1.3.0",
+    "devcert-san": "^0.3.3",
```

Comment:
Here is the problem.  When you merged the changes from the master branch into your fork, it overwrote your changes to `package.json`.  It has reverted back to the original `devcert-san`